src/fetch/transfer_file.py

```diff
@@ -349,21 +349,46 @@ def __str__(self) -> str:
         )
 
 
-def get_transfers(dune: DuneAPI, period: AccountingPeriod) -> list[Transfer]:
-    """Fetches and returns slippage-adjusted Transfers for solver reimbursement"""
+def get_cow_rewards(dune: DuneAPI, period: AccountingPeriod) -> list[Transfer]:
+    """
+    Fetches COW token rewards from orderbook database returning a list of Transfers
+    """
+    # TODO - replace this with results of ORDERBOOK QUERY
     query = DuneQuery.from_environment(
-        raw_sql=open_query("./queries/period_transfers.sql"),
+        raw_sql=open_query("./queries/cow_rewards.sql"),
         network=Network.MAINNET,
-        name="ETH Reimbursement & COW Rewards",
+        name="COW Rewards",
         parameters=[
             QueryParameter.date_type("StartTime", period.start),
             QueryParameter.date_type("EndTime", period.end),
             QueryParameter.number_type("PerBatchReward", COW_PER_BATCH),
             QueryParameter.number_type("PerTradeReward", COW_PER_TRADE),
         ],
     )
-    reimbursements_and_rewards = [Transfer.from_dict(t) for t in dune.fetch(query)]
-    split_transfers = SplitTransfers(period, reimbursements_and_rewards)
+    return [Transfer.from_dict(t) for t in dune.fetch(query)]
+
+
+def get_eth_spent(dune: DuneAPI, period: AccountingPeriod) -> list[Transfer]:
+    """
+    Fetches COW token rewards from orderbook database returning a list of Transfers
```

Comment:
Uses the same method description as `get_cow_rewards`.

Reply:
Copy-pasta! Whoops